src/js/store/factory/marks/Area.ts

```diff
@@ -53,38 +54,33 @@ Area.getHandleStreams = function(area): Map<string, OnEvent[]> {
   const DELTA = sg.DELTA;
   const DX = DELTA + '.x';
   const DY = DELTA + '.y';
-  const streams = {};
+  const streams: {[s: string]: OnEvent[];} = {};
```

Comment:
Minor but might be worth extracting this type definition so that we're consistent across all marks?

Reply:
done in https://github.com/vega/lyra/pull/427/commits/881c1fa64c836a85391926033c5166a5a5c96e47